src/Services/SwaggerService.php

```diff
@@ -817,9 +832,20 @@ public function saveProductionData()
 
     public function getDocFileContent()
     {
-        $documentation = $this->driver->getDocumentation();
+        try {
+            $documentation = $this->driver->getDocumentation();
 
-        $this->openAPIValidator->validate($documentation);
+            $this->openAPIValidator->validate($documentation);
+        } catch (Exception $exception) {
+            $data = $this->generateBaseDataObject();
+
+            $infoConfig = $this->config['info'];
+            $infoConfig['description'] = Arr::get($this->config, 'defaults.error');
+
+            $data['info'] = $this->prepareInfo($infoConfig, ['message' => $exception->getMessage()]);
+
+            return $data;
```

Comment:
please make the following:
- call `generateEmptyData` method here
- modify `generateEmptyData` to able to pass info view and data as args
- generateEmptyData should pass view and data args to the prepare info method

Please check case when package using without the contact email config, seems it will fails with the cycle dependency error, but I not sure